helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/AvroAdapter.java

```diff
@@ -37,6 +37,9 @@
 
   BinaryDecoder newBinaryDecoder(ObjectInput in);
 
+  BinaryDecoder configureBinaryDecoder(byte[] bytes, int offset,
+      int length, BinaryDecoder reuse);
```

Comment:
I recommend aligning the naming with that of the other functions: `newBinaryEncoder`, which also support a `reuse` parameter. It should be valid to pass null in the `reuse` param, and therefore, this function doesn't necessarily only configure an existing decoder, but it is also allowed to create a new one, in some cases.